release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2942,7 +2976,9 @@ It is anticipated that an upcoming z-stream release will include a fix for this
 
 * Creating pods with Microsoft Azure File NFS volumes that are scheduled to the control plane node causes the mount to be denied.
 +
-To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.(link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
+To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes. (link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
+
+* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. Additionally, there is no support for Kuryr installations on {rh-openstack} 17.1. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2228643[*BZ2228643*])
```

Comment:
Sorry I forgot to mention that BZ2228643 belongs to the NFV issue, so my suggestion is:
* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2228643[*BZ2228643*])
* There is no support for Kuryr installations on {rh-openstack} 17.1.

I don't know if there was any reason not to add the "lack of Kuryr support in 17.1" on it's own known issue "bullet"

Reply:
```suggestion
* For clusters that run on {rh-openstack} 17.1 and use network function virtualization (NFV), a known issue in {rh-openstack} prevents succesful cluster deployment. There is no workaround for this issue. Contact Red Hat Support to request a hotfix. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2228643[*BZ2228643*])

*  There is no support for Kuryr installations on {rh-openstack} 17.1. 
```

Reply:
@eurijon Done.